plugins/offers.c

```diff
@@ -317,100 +317,97 @@ static void json_add_offer(struct json_stream *js, const struct tlv_offer *offer
 	struct sha256 offer_id;
 	bool valid = true;
 
-	merkle_tlv(offer->fields, &offer_id);
+	/* FIXME-OFFERS: Rename all fields to offer_ as per spec */
```

Comment:
Is this still pending? Seems like almost all fields got prefixed.

Reply:
Ah, I missed that one in my sweep of "API updates".  Added!